apps/web/components/Layout.tsx

```diff
@@ -40,112 +44,115 @@ export function Layout({
   // TODO: deduplicate me...
   const onlyIntegrationId =
     integrationsRes.data?.length === 1 ? integrationsRes.data[0]?.id : undefined
-
   return (
-    <div className="relative flex h-screen flex-col overflow-y-hidden">
-      <header className="shrink-0 overflow-x-hidden border-b border-gray-100">
-        {/* TODO: Add global control for envName as well as currentUserId in developer mode */}
-        <Container className="h-16 flex-row items-center justify-between py-0">
-          <EnhancedActiveLink
-            href="/"
-            className="shrink truncate text-xl font-bold text-primary">
-            {title}
-          </EnhancedActiveLink>
-
-          {links.length > 0 && (
-            <div className="flex shrink-0 grow items-center justify-end">
-              <nav className="flex space-x-2 text-sm font-medium md:space-x-4">
-                {links.map((l) => (
-                  <EnhancedActiveLink
-                    key={l.href}
-                    href={l.href}
-                    className={twMerge(
-                      'btn btn-sm',
-                      l.primary
-                        ? 'btn-primary link-exact-active:btn-disabled'
-                        : 'btn-ghost link-exact-active:text-black',
-                      !l.fixed && 'hidden md:inline-flex',
-                    )}>
-                    {l.label}
-                  </EnhancedActiveLink>
-                ))}
+    <>
+      <Head>
+        <title>Venice - {title}</title>
```

Comment:
This would produce `Venice - Venice - financial data , fast`

Reply:
True, will fix